docs/architecture/adr-022-system-merge.md

```diff
@@ -0,0 +1,63 @@
+# ADR 022: Streamline system architecture
+
+## Changelog
+
+- 2023/10/19: Initial draft
+
+## Status
+
+Proposed
+
+## Context
+
+The Celestia network is divided into two subsystems owing largely to two different networking implementations. Some of the justification around this decision is as follows:
+
+- Tendermint consensus was battle tested and widely used yet tightly coupled to the networking layer. Using it as close to out of the box as possible meant a massive reduction in engineering resources to reach product launch.
+- Tendermint networking layer was largely unfamiliar to the team and had known frailties. LibP2P was more familiar, had more documentation and community support. It also provided functionality that was not available in Tendermint.
+
+This decision led to the creation of `celestia-node`, a new repository, responsible for data availability sampling and data retrieval. As a consequence of this decision:
+
+- A bridge node was introduced which had to be run side by side with the consensus node, relaying data from the consensus node to the nodes in the DA network (i.e. using LibP2P). This meant:
+  - Extra complexity for node operators
+  - Duplicate storage of squares
```

Comment:
The "duplicate storage of squares" mean duplicates in all 3 nodes i.e., consensus, bridge and data? or just node and consensus?

Reply:
consensus and bridge. A validator is supposed to run both the consensus node and bridge node which means they are keeping two copies of the same data